k8s/prod/common/ia/case-api.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: case-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-case-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-case-case-api
+    version: 0.0.3
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "71f1292c-eadf-4ea9-8444-b82a6b09405a"
+      image: hmctspublic.azurecr.io/ia/case-api:prod-b6b4877a
```

Comment:
@raghera @timja  is imageId taken from the latest existing image in Azure? It looks suspicious because the small hash is the same for all three services...

Reply:
I think Flux replaces the hash with the latest one.  Correct me if I'm wrong @timja

Reply:
it will just put `latest` in or copy the AAT current tag